tests/admin-mode_test.go

```diff
@@ -0,0 +1,270 @@
+			Expect(err).To(BeNil(), "Failed to create chaos experiment")
+			fmt.Println("Pod delete Chaos Experiment Created Successfully")
+
+			//Installing chaos engine for the experiment
+			//Fetching engine file
+			By("Fetching engine file for pod delete experiment")
+			err = exec.Command("wget", "-O", experimentName+"-ce.yaml", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/pod-delete/engine.yaml").Run()
+			Expect(err).To(BeNil(), "Failed to fetch engine file for pod delete")
+
+			//Modify chaos engine spec
+			//Modify Namespace,Name,AppNs,AppLabel of the engine
+			err = exec.Command("sed", "-i",
+				`s/name: nginx-chaos/name: `+engineName+`/g;
+			     s/namespace: default/namespace: `+chaosTypes.ChaosNamespace+`/g;
+			     s/chaosServiceAccount: pod-delete-sa/chaosServiceAccount: litmus-admin/g;
+			     s/jobCleanUpPolicy: 'delete'/jobCleanUpPolicy: 'retain'/g;
+			     s/annotationCheck: 'true'/annotationCheck: 'false'/g;
+			     s/applabel: 'app=nginx'/applabel: 'run=adminapp'/g`,
+				experimentName+"-ce.yaml").Run()
+			//Modify FORCE
+			err = exec.Command("sed", "-i", `/name: FORCE/{n;s/.*/              value: ""/}`, experimentName+"-ce.yaml").Run()
+			Expect(err).To(BeNil(), "Failed to Modify FORCE field of chaos engine")
+
+			//Creating ChaosEngine
+			By("Creating ChaosEngine")
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-ce.yaml", "-n", chaosTypes.ChaosNamespace).Run()
+			Expect(err).To(BeNil(), "Failed to create chaos engine")
+			fmt.Println("ChaosEngine for pod delete created successfully")
+			time.Sleep(2 * time.Second)
+
+			//Fetching the runner pod and Checking if it gets in Running state or not
+			By("Wait for runner pod to come in running sate")
+			runnerNamespace := chaosTypes.ChaosNamespace
+			runnerPodStatus, err := utils.RunnerPodStatus(runnerNamespace, engineName, client)
+			Expect(runnerPodStatus).NotTo(Equal("1"), "Faileded to get in running state")
+			Expect(err).To(BeNil(), "Failed to get the runner pod")
+			fmt.Println("Runner pod for pod delete is in Running state")
+
+			//Waiting for experiment job to get completed
+			//Also Printing the logs of the experiment
+			By("Waiting for job completion")
+			jobNamespace := chaosTypes.ChaosNamespace
+			jobPodLogs, err := utils.JobLogs(experimentName, jobNamespace, engineName, client)
+			Expect(jobPodLogs).To(Equal(0), "Failed to print the logs of the experiment")
+			Expect(err).To(BeNil(), "Failed to get the experiment job pod")
+
+			//Checking the chaosresult
+			By("Checking the chaosresult")
+			chaosResult, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+			Expect(string(chaosResult.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass chaosresult")
+			Expect(err).To(BeNil(), "Failed to get chaosresult")
+		})
+	})
+	// BDD TEST CASE 2
+	// Operator in litmus ns, engine and experiment is in test ns
```

Comment:
The usecase for app and litmus in different ns is already covered  by bdd-1 
The BDD-2 Should test the case where all are in same ns,, normal behavoir
cc: @ksatchit